frontend/src/metabase-lib/lib/metadata/Field.ts

```diff
@@ -77,6 +78,19 @@ class FieldInner extends Base {
     return this.id;
   }
 
+  // `uniqueId` is set by our normalizr schema so it is not always available,
+  // if the Field instance was instantiated outside of an entity
+  getUniqueId() {
+    if (this.uniqueId) {
+      return this.uniqueId;
+    }
+
+    const uniqueId = getUniqueFieldId(this.getId(), this.table_id);
+    this.uniqueId = uniqueId;
+
+    return uniqueId;
+  }
+
```

Comment:
Should we maybe make it a [`getter` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)?